reflex/app.py

```diff
@@ -273,6 +274,7 @@ def add_default_endpoints(self):
         """Add default api endpoints (ping)."""
         # To test the server.
         self.api.get(str(constants.Endpoint.PING))(ping)
+        self.api.post("/_set_cookie/")(set_http_only_cookie)
```

Comment:
I don't believe it is safe/secure to have an API which allows a frontend caller unrestricted access to setup HTTP Only cookies, or at least goes against the security rationale which envisioned HTTP Only cookies in the first place. (I could be wrong)